bin/build_mjml.sh

```diff
@@ -18,11 +18,11 @@ END
 MJML_REGEX="^[^_].+\.mjml$"
 
 if [ "$1" = 'help' -o "$1" = 'h' -o "$1" = '--help' ]; then
-  echo "$USAGE"
+  echo -n "$USAGE\n"
```

Comment:
WTF? 
Do I understand that correctly that you disable default trailing newline and print yours instead? Why would you ever need that?